db/migrate/20160106051255_recreate_people.rb

```diff
@@ -0,0 +1,54 @@
+class RecreatePeople < ActiveRecord::Migration
+  def change
+    # note, wasn't able to just drop and re-add table because of foreign key constraints
+    # (and apparently with postgres, there is no way to temporarily suspend the constraints)
+    # drop_table :people
+    # create_table :people do |t|
+    #   ## base Contact fields
+    #   t.references :division, index: true
+    #   t.string :name   # note agreeing with mel's change of this from 'display_name' as a more consistent field to hold the full name
```

Comment:
That actually wasn't the change I was making.

Based on [this guide](http://www.w3.org/International/questions/qa-personal-names#fielddesign) and many similar resources, I was guessing that 

1. We might not *need* to capture first and last names separately
2. Any suspected reason to do so should be closely examined for alternatives
3. If we really do need it, we should split into given/family names instead of "first" and "last" and not require both

I left it at just "name" at creation time since it would be easy to add extra fields whenever we actually needed them.

Poorly localized name forms based around the assumptions of English-speaking software developers cause a pretty big headache for a lot of people, I'd love for us to not be the cause of headaches on this issue. 

My preference would be for two fields, `name`/`display_name`/`nickname` for what we should call the person and `legal_name`/`official_name` for the purposes of any official/government/financial paperwork. If we need to capture and store given & family name separately, then maybe we should provide a dropdown for populating `legal_name` that gives a few different options for how to compose Given & Family names (I think wordpress does this) or if none are correct allows manual entry.